package-lock.json

```diff
@@ -2736,6 +2736,28 @@
                 }
             }
         },
+        "node_modules/@rollup/plugin-replace": {
```

Comment:
This change is not introduced by this PR. Must be changes that were left uncommitted in the past revisions